plugin.json

```diff
@@ -0,0 +1,88 @@
+                "placeholder": "https://jitsi.example.com"
+            },
+            {
+                "key": "JitsiEmbedded",
+                "display_name": "Embed Jitsi video inside Mattermost",
+                "type": "bool",
+                "help_text": "(Experimental) When true, Jitsi video is embedded as a floating window inside Mattermost."
+            },
+            {
+                "key": "JitsiJWT",
+                "display_name": "Use JWT Authentication for Jitsi",
+                "type": "bool",
+                "help_text": "(Optional) If your Jitsi server uses JSON Web Tokens (JWT) for authentication, set this value to true."
+            },
+            {
+                "key": "JitsiAppID",
+                "display_name": "App ID for JWT Authentication",
+                "type": "text",
+                "help_text": "(Optional) The App ID used for authentication by the Jitsi server and JWT token generator."
+            },
+            {
+                "key": "JitsiAppSecret",
+                "display_name": "App Secret for JWT Authentication",
+                "type": "text",
+                "help_text": "(Optional) The App Secret used for authentication by the Jitsi server and JWT token generator."
+            },
+            {
+                "key": "JitsiLinkValidTime",
+                "display_name": "Meeting Link Expiry Time (minutes)",
+                "type": "number",
+                "help_text": "(Optional) The number of minutes from when the meeting link is created to when it becomes invalid. Minimum is 1 minute. Only applies if using JWT authentication for your Jitsi server.",
+                "default": 30
+            },
+            {
+                "key": "JitsiNamingScheme",
+                "display_name": "Jitsi Meeting Names",
+                "type": "radio",
+                "help_text": "Select how meeting names are generated.",
+                "default": "english-titlecase",
+                "options": [
+                    {
+                        "display_name": "Random English words in title case (e.g. PlayfulDragonsObserveCuriously)",
+                        "value": "english-titlecase"
+                    },
+                    {
+                        "display_name": "UUID (universally unique identifier)",
+                        "value": "uuid"
+                    },
+                    {
+                        "display_name": "Mattermost context specific names. Combination of team name, channel name and random text in public and private channels; personal meeting name in direct and group messages channels.",
+                        "value": "mattermost"
+                    },
+                    {
+                        "display_name": "Allow user to select meeting name",
+                        "value": "ask"
+                    }
+                ]
+            }
```

Comment:
1/5 This seams like a quite important setting to me that should be near the top of the list.

Reply:
@jasonblais what do you think?

Reply:
I'm okay moving it above the JWT auth options, I don't feel strongly about it. I'm not sure how common it is to use JWT auth for Jitsi.

Reply:
I going to move it above the JWT auth options. but in another PR.